### ISSUE TITLE
Random issue

### ISSUE THREAD

**Issue by hespoz** [defold/defold#6877]
### Steps to reproduce

<hr/>

<table>
<tr><td>Build time</td><td>2022-07-11T09:23:08.527760</td></tr>
<tr><td>Defold channel</td><td>editor-alpha</td></tr>
<tr><td>Defold editor sha</td><td>c0ed825ede46b0112d1e4fd744fd50f460073229</td></tr>
<tr><td>Defold engine sha</td><td>80b1b73fd9cdbd4682c2583403fddfbaf0919107</td></tr>
<tr><td>Defold version</td><td>1.3.4</td></tr>
<tr><td>Error</td><td><a href='https://sentry.io/organizations/defold/issues/?query=id%3A"bf3e3bf0c3ab49b69b59ec764c9b432f"'>bf3e3bf0c3ab49b69b59ec764c9b432f</a></td></tr>
<tr><td>GPU</td><td>Intel(R) Iris(TM) Plus Graphics OpenGL Engine</td></tr>
<tr><td>GPU Driver</td><td>2.1 INTEL-16.5.2</td></tr>
<tr><td>Java version</td><td>11.0.15+10-LTS</td></tr>
<tr><td>OS arch</td><td>x86_64</td></tr>
<tr><td>OS name</td><td>Mac OS X</td></tr>
<tr><td>OS version</td><td>11.6</td></tr>
<table>

**Comment by britzl** [comment]
```
com.sun.glass.ui.View$2.flush.class java.lang.UnsupportedOperationException: Flush is forbidden from target!
    at com.sun.glass.ui.View$2.flush(View.java:1017)
    at com.sun.javafx.tk.quantum.QuantumClipboard.flush(QuantumClipboard.java:276)
    at com.sun.javafx.tk.quantum.QuantumToolkit.startDrag(QuantumToolkit.java:1301)
    at javafx.scene.Scene$DnDGesture.dragDetectedProcessed(Scene.java:3066)
    at javafx.scene.Scene$DnDGesture.process(Scene.java:3135)
    at javafx.scene.Scene$MouseHandler.process(Scene.java:3897)
    at javafx.scene.Scene.processMouseEvent(Scene.java:1874)
    at javafx.scene.Scene$ScenePeerListener.mouseEvent(Scene.java:2607)
    at com.sun.javafx.tk.quantum.GlassViewEventHandler$MouseEventNotification.run(GlassViewEventHandler.java:411)
    at com.sun.javafx.tk.quantum.GlassViewEventHandler$MouseEventNotification.run(GlassViewEventHandler.java:301)
    at java.security.AccessController.doPrivileged(AccessController.java)
    at com.sun.javafx.tk.quantum.GlassViewEventHandler.lambda$handleMouseEvent$2(GlassViewEventHandler.java:450)
    at com.sun.javafx.tk.quantum.QuantumToolkit.runWithoutRenderLock(QuantumToolkit.java:424)
    at com.sun.javafx.tk.quantum.GlassViewEventHandler.handleMouseEvent(GlassViewEventHandler.java:449)
    at com.sun.glass.ui.View.handleMouseEvent(View.java:551)
    at com.sun.glass.ui.View.notifyMouse(View.java:937)
    at com.sun.glass.ui.mac.MacView.notifyMouse(MacView.java:127)
```

**Comment by britzl** [comment]
@hespoz when does this happen? Can you reproduce the issue?

**Comment by hespoz** [comment]
It happen when I was importing images using drag and drop from my filesystem to one of the folders inside my project, it happen two times already, but not sure if I can reproduce it. Let me try again.

**Comment by hespoz** [comment]
@britzl btw It never happen again, I will close it if you don't mind. Thanks
